Assets/Scripts/WheelMotor.cs

```diff
@@ -25,6 +29,11 @@ private void FixedUpdate()
         UpdatePower();
         UpdatePosition();
         Render();
+        if (Math.Abs(CurrentPower) <= 0.05) {
```

Comment:
Super nitpicky, but we should use `Mathf.Abs` since it's already imported, instead importing `System` and using `Math`